### PR TITLE
Improvements

### LINKED ISSUE #4: Add some deployment examples
Add some very basic deployment examples:

* Deploy in old-timey (and reliable) vixie-cron-esque environment
* Deploy as kubernetes cronjob
* Lambda
* ..

### PR DESCRIPTION
**Pull request by gianrubio**
@pieterlange 
Fix #4

### CODE REVIEW

**Comment by pieterlange** [comment]
Looks very good to me :+1:

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,4 @@
 FROM alpine:3.5
```
Comment (pieterlange) [inline Dockerfile:2]: nit: Create and run as non-privileged user here ?

Comment (gianrubio) [reply Dockerfile:2]: done, could you review it again?

**Comment by pieterlange** [comment]
Why run multiple database backups in the same job? Just schedule a run for each database? :)

**Comment by gianrubio** [comment]
> Why run multiple database backups in the same job? Just schedule a run for each database? :)

I rollback to just one backup per job

**Pull request merged**